source/sort.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const localeCompare = (strArray) => strArray.sort( (a, b) => a.localeCompare(b) ); // сортировка по алфавиту
```

Comment:
```suggestion
const localeCompare = (a, b) => a.localeCompare(b); // сортировка по алфавиту
```
давай так - а ниже явно sort вызывать - чтобы не получалось что большой кусок кода обернут в `localeCompare(arr.blablabla)`  и было `arr.blablabla.sort(localeCompare)`

Reply:
тогда непонятно какие a, b передавать в localeCompare